Rakefile

```diff
@@ -8,3 +8,79 @@ Rake::TestTask.new do |t|
   t.test_files = FileList['test/**/*test.rb']
   t.verbose = true
 end
+
+def print(family, template, distro, major, minor, distro_prefix, release, validate, ipv4_subnet, ipv6_subnet)
+  renderer = TemplateRenderer.new(family, template)
+  code, stdout, stderr, contents = renderer.render(distro, major, minor, distro_prefix, release, validate, ipv4_subnet, ipv6_subnet)
+  puts contents
+  puts stdout
+  puts stderr
+  code
+end
+
+$LOAD_PATH << "test"
+require "test_helper"
+desc "Render a selected template"
+namespace :render do
+  namespace :redhat8 do
+    namespace :dhcp do
+      namespace :ipv4_only do
+        task :provision do
+          print('Redhat', 'provisioning_templates/provision/kickstart_default.erb', "CentOS", 8, 0, "RHEL", nil, true, :dhcp, :none)
+        end
+
+        task :pxelinux do
+          print('Redhat', 'provisioning_templates/PXELinux/kickstart_default_pxelinux.erb', "CentOS", 8, 0, "RHEL", nil, false, :dhcp, :none)
+        end
+      end
+      namespace :ipv6_only do
+        task :provision do
+          print('Redhat', 'provisioning_templates/provision/kickstart_default.erb', "CentOS", 8, 0, "RHEL", nil, true, :none, :dhcp)
+        end
+
+        task :pxelinux do
+          print('Redhat', 'provisioning_templates/PXELinux/kickstart_default_pxelinux.erb', "CentOS", 8, 0, "RHEL", nil, false, :none, :dhcp)
+        end
+      end
+      namespace :dualstack do
+        task :provision do
+          print('Redhat', 'provisioning_templates/provision/kickstart_default.erb', "CentOS", 8, 0, "RHEL", nil, true, :dhcp, :dhcp)
+        end
+
+        task :pxelinux do
+          print('Redhat', 'provisioning_templates/PXELinux/kickstart_default_pxelinux.erb', "CentOS", 8, 0, "RHEL", nil, false, :dhcp, :dhcp)
+        end
+      end
+    end
+
+    namespace :static do
```

Comment:
Looks like static is within the DHCP namespace?

Reply:
Not in my editor, no.

Reply:
Hmm, I was trusting Github but I've seen some issue with alignment and expanding in the past so that's probably ok then.